src/components/custom/_dbanner.tsx

```diff
@@ -1,12 +1,17 @@
 import React from 'react'
 import styled from 'styled-components'
-import PropTypes from 'prop-types'
 import { Flex } from 'components/containers'
 import { Header, QueryImage } from 'components/elements'
 import { LinkButton } from 'components/form'
 import { localize } from 'components/localization'
 import device from 'themes/device'
 
+type DBannerProps = {
+    background_pattern?: string
+    data: object
```

Comment:
Can we specify type of properties of the object instead of using `object` type?

Reply:
yes, will update the PR.